internal/database/database.go

```diff
@@ -152,9 +152,30 @@ func (dbs *SQLdb) getFiles(datasetID string) ([]*FileInfo, error) {
 	files := []*FileInfo{}
 	db := dbs.DB
 
-	const query = "SELECT a.file_id, dataset_id, display_file_name, file_name, file_size, " +
-		"decrypted_file_size, decrypted_file_checksum, decrypted_file_checksum_type, file_status from " +
-		"local_ega_ebi.file a, local_ega_ebi.file_dataset b WHERE dataset_id = $1 AND a.file_id=b.file_id;"
+	const query = `
+		SELECT f.stable_id AS file_id,
```

Comment:
how about we have a view with this kind of information in the db, to make it easier to do get this information ?

Reply:
i think we should do the changes now, instead of postponing, as the code for the db will get merged in that repo either way. I don't think they will conflict in any way.

Reply:
Besides the "juggling different PRs" aspect, I also believe it will easier to do now and less risks for mistakes.

Reply:
Sure! :) it's gonna be outside the scope of this PR, but feel free to start sketching something out!